app/locales/en.json

```diff
@@ -7,5 +7,170 @@
+  "Healthy": "Healthy",
+  "Has symptoms": "Has symptoms",
+  "Tested positive": "Tested positive",
+  "Zip code information ": "Zip code information ",
+  "Join us in the most important crowdsource!": "Join us in the most important crowdsource!",
+  "Make your voluntary contribution to the health care system and register your health condition now. Together we can create an overview.": "Make your voluntary contribution to the health care system and register your health condition now. Together we can create an overview.",
+  "Share on": "Share on",
+  "NB!": "NB!",
+  "This page is currently not affiliated with the authorities in any way.": "This page is currently not affiliated with the authorities in any way.",
+  "Read how the data you enter is processed here!": "Read how the data you enter is processed here!",
+  "Use the link you noted down last time to update your health condition": "Use the link you noted down last time to update your health condition.",
+  "Unreported cases": "Unreported cases",
+  "The spread of COVID-19 in our country is unknown. Help us create a better overview.": "The spread of COVID-19 in our country is unknown. Help us create a better overview.",
+  "Contribute anonymously": "Contribute anonymously",
+  "Report your health condition anonymously": "Report your health condition anonymously",
+  "Insight for everyone": "Insight for everyone",
+  "The data is open to everyone. Look at maps and graphs, or download the data and create your own visualizations.": "The data is open to everyone. Look at maps and graphs, or download the data and create your own visualizations.",
+  "Everything is open": "Everything is open",
+  "The website is open source and built by volunteers. Feel free to join.": "The website is open source and built by volunteers. Feel free to join.",
+  "Total reports": "Total reports",
+  "Infected / Have been tested": "Infected / Have been tested",
+  "In close contact with someone who's infected": "IIn close contact with someone who's infected",
+  "Has symptoms": "Has symptoms",
+  "You must approve the privacy statement to report your data.": "You must approve the privacy statement to report your data.",
+  "Your state of health": "Your state of health",
+  "Report your health condition": "Report your health condition",
+  "Thank you for your contribution! Use this link if your health condition changes:": "Thank you for your contribution! Use this link if your health condition changes:",
+  "Join the most important crowdsource! Regardless if you're healthy or not, please submit the form below – that is also valuable information! ": "Join the most important crowdsource! Regardless if you're healthy or not, please submit the form below – that is also valuable information! ",
+  "Geography and demography": "Geography and demography",
+  "/healthcondition/": "/healthcondition/",
```

Comment:
I don't think we need this after https://github.com/BustByte/coronastatus/pull/199

Reply:
yep, I made sure the templates use the variables. Removed!